ddtrace/llmobs/_utils.py

```diff
@@ -3,11 +3,35 @@
 from ddtrace import Span
 from ddtrace import config
 from ddtrace.ext import SpanTypes
+from ddtrace.llmobs._constants import INPUT_DOCUMENTS
+from ddtrace.llmobs._constants import INPUT_MESSAGES
+from ddtrace.llmobs._constants import INPUT_VALUE
 from ddtrace.llmobs._constants import LANGCHAIN_APM_SPAN_NAME
 from ddtrace.llmobs._constants import ML_APP
+from ddtrace.llmobs._constants import OUTPUT_DOCUMENTS
+from ddtrace.llmobs._constants import OUTPUT_MESSAGES
+from ddtrace.llmobs._constants import OUTPUT_VALUE
 from ddtrace.llmobs._constants import SESSION_ID
 
 
+def _get_input_tag_key_from_span_kind(span_kind):
+    if span_kind == "llm":
+        return INPUT_MESSAGES
+    elif span_kind == "embedding" or "retrieval":
+        return INPUT_DOCUMENTS
```

Comment:
Not sure if we need this right now as embeddings/retrieval spans shouldn't be relevant to this PR.
```suggestion
```